contentcuration/contentcuration/utils/i18n.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+import importlib
+import io
+import json
+import os
+
+from django.utils.translation import get_language
+from django.utils.translation import get_language_bidi
+from django.utils.translation import get_language_info
+
+import contentcuration
+
+
+# List of intl codes that Studio officially supports
+SUPPORTED_LANGUAGES = [
+    "ar",
+    "en",
+    "es-es",
+    "fr-fr",
+]
+
+
+def get_installed_app_locale_path(appname):
```

Comment:
I don't see this referenced anywhere, can remove possibly?

Reply:
Ah I kept this deliberately thinking that it might be helpful when we go back to do another pass on `kolibri-tools` if we want to break each of the SPAs into their own self-contained i18n string groups. Also - looking at it again - that might ultimately play out entirely differently than how this fn expects since Studio's solution to that won't use the same configuration as Kolibri.

Will remove - will know where to find it if/when we want to borrow it in the future.